initramfs/scripts/bootloader-script

```diff
@@ -0,0 +1,190 @@
+# -*- shell-script -*- vim:ft=sh:
+#---------------------------------------------------------------------
+# Description: Update kernel/core snap.
+# Entry point: mountroot().
+#---------------------------------------------------------------------
+
+pre_mountroot()
```

Comment:
This and the following get_partition_from_label looks like (almost) verbatim copies from the script in ubuntu-core-rootfs. I wonder if we can share code here?